package.json

```diff
@@ -132,6 +132,8 @@
     "jsdom-screenshot": "^3.2.0",
     "karma": "^5.1.0",
     "karma-chrome-launcher": "^3.1.0",
+    "karma-coverage-istanbul-reporter": "^3.0.3",
+    "@jsdevtools/coverage-istanbul-loader": "^3.0.3",
```

Comment:
Packages should be listed alphabetically in this file

Reply:
Prettier doesn't do that?

Reply:
`npm` does this usually automatically when doing `npm install` or similar. Prettier thus leaves that alone I think